src/ILCompiler.ReadyToRun/src/ILCompiler.ReadyToRun.csproj

```diff
@@ -153,6 +155,10 @@
     <Compile Include="Compiler\DependencyAnalysis\ReadyToRunSymbolNodeFactory.cs" />
     <Compile Include="Compiler\DependencyAnalysis\TypeAndMethod.cs" />
     <Compile Include="Compiler\IRootingServiceProvider.cs" />
+    <Compile Include="IBC\ibcdatamodel.cs" />
```

Comment:
Could these three files be PascalCased like everything else?

Reply:
Yes, done